samples/highcharts/demo/flame/demo.details

```diff
@@ -0,0 +1,12 @@
+---
+name: Flame chart
+authors:
+  - Kacper Madej
+js_wrap: b
+alt_text: >-
+  Highcharts flame chart JavaScript example graph shows
+  a chart rendering process as stacked tasks and subtasks.
+tags: []
+categories:
+  - More chart types
```

Comment:
We need to add it to `index.htm` also, the new system is not in production yet.

Reply:
Added.